src/albums/albums.controller.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Put,
+  Param,
+  Delete,
+  UseInterceptors,
+  ClassSerializerInterceptor,
+  HttpCode,
+  ParseUUIDPipe,
+  NotFoundException,
+  InternalServerErrorException,
+} from '@nestjs/common';
+import {
+  ApiOperation,
+  ApiTags,
+  ApiCreatedResponse,
+  ApiOkResponse,
+  ApiBadRequestResponse,
+  ApiNotFoundResponse,
+  ApiParam,
+  ApiNoContentResponse,
+} from '@nestjs/swagger';
+import { throwException } from 'src/utils/helpers';
+import { AlbumsService } from './albums.service';
+import { CreateAlbumDto } from './dto/create-album.dto';
+import { UpdateAlbumDto } from './dto/update-album.dto';
+import { AlbumEntity } from './entities/album.entity';
+
+@UseInterceptors(ClassSerializerInterceptor)
```

Comment:
Could not find where "ClassSerializerInterceptor" is used in albums module.
Could you please point to its usage?

Reply:
Seems to be I have only used it in user.entity.ts and "reserved for future".